tests/tests/swfs/avm2/date/Test.as

```diff
@@ -0,0 +1,35 @@
+﻿package {
+	public class Test {
+	}
+}
+
```

Comment:
For AVM1, we have a fairly in-depth test suite for each getter/setter pair, the constructor, `UTC`, and so on. I would highly recommend porting over those tests to AS3 as both objects are similar and implementing them is fairly subtle. We'd also get a good idea of what is similar and what differs between AS2 and AS3 `Date`.